src/portable/st/stm32_fsdev/dcd_stm32_fsdev_pvt_st.h

```diff
@@ -98,6 +98,11 @@
   #undef USB_PMAADDR
   #define USB_PMAADDR USB1_PMAADDR
 
+// NOTE(lbayes): Added the following block for L4 support
```

Comment:
please remove the Note line, also use the `OPT_MCU_STM32L4` macro instead of number 309. Since L4 has two series with different usb controller. You may want to make sure it is L4x2 or L4x3. (L4x5,x6 use dwc2).